packages/discord.js/src/structures/CommandInteraction.js

```diff
@@ -145,15 +159,23 @@ class CommandInteraction extends BaseInteraction {
   }
 
   // These are here only for documentation purposes - they are implemented by InteractionResponses
-  /* eslint-disable no-empty-function */
+
   deferReply() {}
+
   reply() {}
+
   fetchReply() {}
+
   editReply() {}
+
   deleteReply() {}
+
   followUp() {}
+
   launchActivity() {}
+
   showModal() {}
+
   awaitModalSubmit() {}
```

Comment:
I've seen other methods like this modified to have async. Should these have it too?